libxdmcp will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxdmcp it will fail in the following way:

```
2024/11/12 23:12:40 INFO running step "test/pkgconf"
2024/11/12 23:12:40 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:40 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:40 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:40 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:40 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + basename /home/build/melange-out/libxdmcp-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + dev_pkg=libxdmcp-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + apk info -L libxdmcp-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + grep -q ^Name: usr/lib/pkgconfig/xdmcp.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + basename usr/lib/pkgconfig/xdmcp.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + lib_name=xdmcp uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + grep -q '^usr/lib/pkgconfig/xdmcp.pc$\|^usr/share/pkgconfig/xdmcp.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + echo usr/lib/pkgconfig/xdmcp.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + pkgconf --exists xdmcp uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN + pkgconf --print-errors --exists xdmcp uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN Package xproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN Perhaps you should add the directory containing `xproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 WARN Package 'xproto', required by 'xdmcp', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:40 INFO pod 70a58a07debc501768e61ae9fe65e1e7e0eb313fe9018ee93eac52f1bf4c4cd6 terminated
2024/11/12 23:12:40 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:40 INFO   workspace dir: /home/user/tmp/melange-workspace-3402574751
2024/11/12 23:12:40 INFO   guest dir: /home/user/tmp/melange-guest-4252101810
2024/11/12 23:12:40 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxdmcp] Error 1
```

--- a/libxdmcp.yaml
+++ b/libxdmcp.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxdmcp
   version: 1.1.5
-  epoch: 3
+  epoch: 4
   description: X11 Display Manager Control Protocol library
   copyright:
     - license: MIT
@@ -52,7 +52,11 @@ subpackages:
     dependencies:
       runtime:
         - libxdmcp
+        - xorgproto
     description: libxdmcp dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

